app/src/main/res/drawable/card.xml

```diff
@@ -4,10 +4,10 @@
     <item>
         <shape>
             <padding
-                android:top="8dp"
-                android:right="8dp"
-                android:bottom="5dp"
-                android:left="8dp"/>
+                android:top="0dp"
```

Comment:
+1 to the bottom padding if it cuts off the shadow. I'm wary of the side padding though since it's pretty inconsistent between different widgets and launchers. Maybe @kabouzeid has some input on that.